ios/Runner/AppDelegate.swift

```diff
@@ -6,29 +6,19 @@ import flutter_local_notifications
 
 @UIApplicationMain
 @objc class AppDelegate: FlutterAppDelegate {
-    
-    /// Registers all pubspec-referenced Flutter plugins in the given registry.
-    static func registerPlugins(with registry: FlutterPluginRegistry) {
```

Comment:
Why we remove it?

Reply:
We previously created this function for reuse, but it is quite confusing and unnecessary. We will register directly for each Plugin when used